lib/aws-manager.js

```diff
@@ -1122,39 +1025,30 @@ class AwsManager {
     
     let keyName = this.createKeyPairName(workerName);
 
-    if (this.__knownKeyPairs.includes(keyName)) {
+    if (_.includes(this.__knownKeyPairs, keyName)) {
       // Short circuit checking for a key but return
       // a promise so this cache is invisible to the
       // calling function from a non-cached instance
       return;
     }
 
-    let res = await this.ec2.describeKeyPairs.inRegions(this.ec2.regions, {
-      Filters: [
-        {
-          Name: 'key-name',
-          Values: [keyName],
-        },
-      ],
-    });
+    for (let region of _.keys(this.ec2)) {
```

Comment:
again, ++ for making this simpler... I mean really this is actually simpler because the response format is less convoluted...

Nit:
```js
await Promise.all(_.map(this.ec2, async (ec2, region) => {
   let keyPairs = await ec2.describeKeyPairs({
     ...
   }).promise();
   
   if (keyPairs.KeyPairs[0]) {
     return;
   }

   await ec2.importKeyPair({
     ....
   }).promise();
}));
```
This would give you max concurrency... and a slight complexity reduction as you won't have to do `this.ec2[region]` anymore... because `_.map` gives us both value and key.
(yes, lodash ordering of `(value, key) => {` in map, forEach etc... is driving me insane :)